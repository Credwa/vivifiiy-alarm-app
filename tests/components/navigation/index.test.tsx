import React from 'react';
import renderer from 'react-test-renderer';
import Navigation from '@/components/navigation';

jest.useFakeTimers();
describe('<Navigation />', () => {
  it(`renders correctly`, () => {
    const tree = renderer.create(<Navigation />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
