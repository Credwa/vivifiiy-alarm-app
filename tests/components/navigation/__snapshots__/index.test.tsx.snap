// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navigation /> renders correctly 1`] = `null`;
